packages/client/hmi-client/src/page/project/components/tera-project-configuration-modal.vue

```diff
@@ -111,20 +112,19 @@ const props = defineProps<{
 
 const emit = defineEmits(['close-modal']);
 
-const auth = useAuthStore();
 const router = useRouter();
-const userId = auth.user?.id ?? '';
 
 const title = ref(props.project?.name ?? '');
+const isTitleInvalid = computed(() => isEmpty(title.value));
```

Comment:
What do we want to enforce here? So `""` is invalid, but `" "` is okay?

Is title being empty `""` causing problems?

Reply:
It's not, but the UI is not made for, which creates a lot of glitch.
This is a basic test, and if someone really wants to have a name equal one space, their prerogative.